learning/step_by_step/your_first_game.rst

```diff
@@ -0,0 +1,768 @@
+        $ScoreTimer.stop()
+        $MobTimer.stop()
+
+Now connect the ``timeout()`` signal of each of the Timer nodes.
+``StartTimer`` will start the other two timers. ``ScoreTimer`` will
+increment the score by 1.
+
+::
+
+    func _on_StartTimer_timeout():
+        $MobTimer.start()
+        $ScoreTimer.start()
+
+    func _on_ScoreTimer_timeout():
+        score += 1
+
+In ``_on_MobTimer_timeout()`` we will create a mob instance, pick a
+random starting location along the ``Path2D``, and set the mob in
+motion. The ``PathFollow2D`` node will automatically rotate as it
+follows the path, so we will use that to select the mob's direction as 
+well as its position.
+
+Note that a new instance must be added to the scene using
+``add_child()``.
+
+::
+
+    func _on_MobTimer_timeout():
+        # choose a random location on the Path2D
+        $"MobPath/MobSpawnLocation".set_offset(randi())
+        # create a Mob instance and add it to the scene
+        var mob = Mob.instance()
+        add_child(mob)
+        # choose a direction and position
+        var direction = $"MobPath/MobSpawnLocation".rotation
+        mob.position = $"MobPath/MobSpawnLocation".position
+        # add some randomness to the direction
+        direction += rand_range(-PI/4, PI/4)
+        # textures are oriented pointing up, so add 90 degrees
+        mob.rotation = direction + PI/2
+        # choose the velocity
+        mob.set_linear_velocity(Vector2(rand_range(mob.MIN_SPEED, mob.MAX_SPEED), 0).rotated(direction))
+
+.. important:: In functions requiring angles, GDScript uses *radians*, 
+               not degrees. If you're more comfortable working with 
+               degrees, you'll need to use the ``deg2rad()`` and 
+               ``rad2deg()`` functions to convert between the two measures.
+
+HUD
+---
+
+The final piece our game needs is a UI: an interface to display things
+like score, a "game over" message, and a restart button. Create a new
+scene, and add a ``CanvasLayer`` node named ``HUD`` ("HUD" stands for
+"heads-up display", meaning an informational display that appears as an
+overlay, on top of the game view).
+
+The HUD is going to display the following information:
+
+-  Score (changed by ``ScoreTimer``)
+-  A message (i.e. "Game Over" or "Get Ready!")
+-  A "Start" button to begin the game
+
+Create the following children of the ``HUD`` node:
+
+-  :ref:`Label <class_Label>` (named ``ScoreLabel``)
+-  :ref:`Label <class_Label>` (named ``MessageLabel``)
+-  :ref:`Button <class_Button>` (named ``StartButton``)
+-  :ref:`Timer <class_Timer>` (named ``MessageTimer``)
+
+.. note:: **Anchors and Margins** ``Control`` nodes have a position and size,
+          but they also have an anchors and margins. Anchors define the
+          origin, or the reference point for the edges of the node. Margins
+          update automatically when you move or resize a control node. They
+          represent the distance from the control node’s edges to its anchor.
```

Comment:
> node’s

This apostrophe looks different from the rest. 😛

Reply:
Good catch - that's a subtle one...